torch/csrc/autograd/profiler.h

```diff
@@ -150,6 +185,12 @@ struct TORCH_API Event final {
     }
     throw std::runtime_error("unknown EventKind");
   }
+
+  // Get enum kind of this event.
+  EventKind eventKind() const {
```

Comment:
nit: just kind()

Reply:
`kind()` is already the name of the above function that returns a string representation of the enum.

Reply:
ah, right; to be honest we'd better call the function above as kind_str or smth similar, but we can refactor later